core/src/test/scala/unit/kafka/server/ListOffsetsRequestTest.scala

```diff
@@ -22,18 +22,27 @@ import org.apache.kafka.common.message.ListOffsetsResponseData.ListOffsetsPartit
 import org.apache.kafka.common.protocol.{ApiKeys, Errors}
 import org.apache.kafka.common.requests.{ListOffsetsRequest, ListOffsetsResponse}
 import org.apache.kafka.common.{IsolationLevel, TopicPartition}
+import org.apache.kafka.server.config.ServerConfigs
 import org.junit.jupiter.api.Assertions._
 import org.junit.jupiter.params.ParameterizedTest
 import org.junit.jupiter.params.provider.ValueSource
 
-import java.util.Optional
+import java.util.{Optional, Properties}
+import scala.collection.Seq
```

Comment:
This seems unused?

Reply:
`Seq` and `Properties` are both used here. What do you think is unused?

Reply:
I've seen it where the IDE marks it as unused but if you remove it, it won't build. Not sure if that's what is happening here.